README.md

```diff
@@ -1,106 +1,170 @@
-</properties>
+  </properties>
 ```
 
-3. Run `mvn install` to install packages and their dependencies locally.
+3. Run `mvn install -DskipTests` in this repo's directory to install Conjur API into your
+   local maven repository.
 
-If generating a JAR is preferred, you can build the library locally and add the dependency to the project manually by following the setup steps below:
+### Using the Jarfile
 
-1. Clone the Conjur Java API repo locally: `git clone {repo}`
-2. `cd conjur-api-java`
-3. Run `mvn package -DskipTests` to generate a JAR file. These files should be in the `target` directory of the repo
-    
-    _NOTE:_ we ran `mvn package` without running the integration tests, since these require access to a Conjur instance. You can run the integration tests with mvn package once you finished with the configuration. For more information on how to run the tests, take a look at our [Contributing](https://github.com/cyberark/conjur-api-java/blob/master/CONTRIBUTING.md) guide.
+If generating a JAR is preferred, you can build the library locally and add the dependency
+to the project manually by following the setup steps below:
 
-4. Follow the steps outlined [here](https://www.jetbrains.com/help/idea/library.html) for information on how to add JAR files into the new app's project files using Intellij and [here](https://help.eclipse.org/kepler/index.jsp?topic=%2Forg.eclipse.wst.webtools.doc.user%2Ftopics%2Ftwplib.html) for Eclipse
+1. Clone the Conjur Java API repo locally: `git clone {repo}`
+2. Go into the cloned repository with `cd conjur-api-java`
+3. Run `mvn package -DskipTests to generate a JAR file. The output `.jar` files will be located
+   in the `target` directory of the repo
+
+_NOTE:_ we ran `mvn package` without running the integration tests, since these require access
+to a Conjur instance. You can run the integration tests with mvn package once you finished with
+the configuration. For more information on how to run the tests, take a look at our
+[Contributing](https://github.com/cyberark/conjur-api-java/blob/master/CONTRIBUTING.md) guide.
+
+4. Follow the steps outlined [here](https://www.jetbrains.com/help/idea/library.html) for
+   information on how to add JAR files into the new app's project files using Intellij and
+   for Eclipse you can do:
+   `Right click project > Build Path > Configure Build Path > Library > Add External JARs`.
+5. If you are working with the CLI, you can manually install the `.jar` into your Maven
+   repository by running the following but replacing `$VERSION` with the appropriate version
+   of the API:
+   ```sh-session
+   $ mvn org.apache.maven.plugins:maven-install-plugin:2.5.2:install-file \
+       -Dfile=/path/to/api/repo/target/conjur-api-$VERSION.jar
+
+   $ # or
+
+   $ mvn org.apache.maven.plugins:maven-install-plugin:2.5.2:install-file \
+       -Dfile=/path/to/api/repo/target/conjur-api-$VERSION-with-dependencies.jar
+
+   $ or
```

Comment:
```suggestion
   $ # or
```
not gonna lie this layout is weird but at least we should be consistent

Reply:
I split these up into individual code blocks too